tox.ini

```diff
@@ -8,7 +8,8 @@ commands =
 deps=
     pytest
     greenlet
-    redbaron
+    redbaron==0.6.2
```

Comment:
Adding your dependencies here should not be necessary. Tox can install your pkg.